packages/core/src/utils/security/handlers/apiKey.ts

```diff
@@ -0,0 +1,38 @@
+import { fromNullable, getOrElse, mapNullable } from 'fp-ts/lib/Option';
+import { pipe } from 'fp-ts/lib/pipeable';
+import { get } from 'lodash';
+import { SecurityScheme } from './types';
+import { when } from './utils';
+
+export const apiKeyInCookie = {
+  test: ({ type, in: where }: SecurityScheme) => where === 'cookie' && type === 'apiKey',
+  handle: <R, I>(someInput: I, name: string, resource?: R) => {
+    const probablyCookie = get(someInput, ['headers', 'cookie']);
```

Comment:
I like this "probably" naming convention :)

Reply:
well, you know, "life was like a box of chocolates. You never know what you're gonna get." ;)